### PR TITLE
Refactor installers

### PR DESCRIPTION
**Pull request by epsstan**
This PR refactors the various installers as follows:

1/ Reduce the # of installers - Delete the AWS installer. We can standardize on Docker demo/tutorial purposes. We can later add cloud provider tailored installers later 

2/ Simplify the installers - The Docker Compose installer shows how Legend can be used with Gitlab.com. Running a Gitlab instance as a container was adding additional complexity wrto. creating SSL certificates etc. While this is required for a "production" installation, this is also organization specific.  Users of Legend can install/configure a Gitlab instance as they see fit.

### CODE REVIEW

File: installers/docker-compose-remote-gitlab/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+This directory contains a minimal Docker Compose specification that spins up an "instance of Legend" with the following components
+
+* Legend Studio 
+* Legend Engine 
+* Legend SDLC 
+
+All three components are wired to use a remote Gitlab instance (gitlab.com).
+
+# Usage Instructions
```
Comment (nsprimmer) [inline installers/docker-compose-remote-gitlab/README.md:11]: Should be "##"

File: installers/docker-compose-remote-gitlab/part2.README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+### 1. Configure the depot server 
+
+No special configuration required for the depot server.
+
+### 2. Read indexed artifacts 
+
+Once artifacts have been indexed by the depot store server, the depot server's APIs serve up these entities. 
+
+For e.g entities for a version can be fetched using the following API
```
Comment (nsprimmer) [inline installers/docker-compose-remote-gitlab/part2.README.md:81]: Another place we should add a Swagger link

File: installers/docker-compose/depot-store/config/authorisedIdentities.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "Artifacts": ["${DEPOT_STORE_ADMIN_USER}"],
```
Comment (akphi) [inline installers/docker-compose/depot-store/config/authorisedIdentities.json:2]: Do we really need this? I have setup `depot-store` before without needing this

File: installers/docker-compose/depot-store/config/settings.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    <offline>false</offline>
+    <profiles>
+        <profile>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <repositories>
+                <repository>
+                    <id>13480401</id>
+                    <url>https://gitlab.com/api/v4/groups/13480401/-/packages/maven</url>
```
Comment (akphi) [inline installers/docker-compose/depot-store/config/settings.xml:36]: Should we hardcode all of these?

**Review by akphi — APPROVED** [review]

**Pull request merged**